kafl_fuzzer/common/config/default_settings.yaml

```diff
@@ -0,0 +1,25 @@
+# kAFL default configuration
+
+# general options
+processes: 1
+
+# fuzz options
+cpu_offset: 0
```

Comment:
so, you have cpu_offset:0 here but not verbose: false? :-)

Reply:
I don't understand what you mean here specifically regarding the relationship between `cpu_offset` and `verbose` ?

Reply:
just wondering where you draw the line between adding the default to settings.yaml or hardcoding it. Nevermind..

Reply:
So, I draw the line beyond boolean values, which are `False` by default and easy to understand.
All the rest should be explicit I believe